packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.test.tsx

```diff
@@ -174,7 +174,7 @@ describe('VitalsBiometricsForm: ', () => {
       message: 'Internal Server Error',
       response: {
         status: 500,
-        statusText: 'Internal Server Error',
+        statusText: t('checkForValidity', 'Some of the values entered are invalid'),
```

Comment:
```suggestion
        statusText: 'Some of the values entered are invalid',
```
Reverting this, we actually don't need it in the test, sorry!